src/battle/tutorial_battle.cpp

```diff
@@ -0,0 +1,140 @@
+				 "Minotaur, main character hits!"
+			  << std::endl;
+	std::cout << "The main character did 6 to the Minotaur" << std::endl;
+	std::cout << "The Minotaur's HP is now 70" << std::endl;
+	std::cout << "Minotaur attacks!" << std::endl;
+	std::cout << "Minotaur rolled 1" << std::endl;
+	std::cout << "The main character has better AC than the attack of the "
+				 "Minotaur, Minotaur misses!"
+			  << std::endl;
+	std::cout << std::endl;
+
+	std::cout << "Please select the Attack option:" << std::endl;
+	std::cout << "1. Attack" << std::endl;
+	std::cout << "2. dodge" << std::endl;
+	std::cout << "3. flee" << std::endl;
+
+	utils::GetUserInput(tutorial_choice);
+	std::cout << std::endl;
+	while (tutorial_choice != 1)
+	{
+		std::cout << "Invalid choice! Please choose again: " << std::endl;
+		utils::GetUserInput(tutorial_choice);
+	}
+	std::cout << "Main character attacks!" << std::endl;
+	std::cout << "The main character rolled 20, it's a critical your damage "
+				 "roll will be doubled"
+			  << std::endl;
+	std::cout << "The main character has better roll than the AC of the "
+				 "Minotaur, main character hits!"
+			  << std::endl;
+	std::cout << "The main character did 15 to the Minotaur" << std::endl;
+	std::cout << "The Minotaur's HP is now 55" << std::endl;
+	std::cout << "Minotaur attacks!" << std::endl;
+	std::cout << "Minotaur rolled 16" << std::endl;
+	std::cout << "The Minotaur has better roll than the AC of the "
+				 "main character, Minotaur hits!"
+			  << std::endl;
+	std::cout << "The Minotaur did 8 to the main character" << std::endl;
+	std::cout << "The main character's HP is now 2" << std::endl;
+	std::cout << std::endl;
+
+	std::cout << "Please select the dodge option:" << std::endl;
+	std::cout << "1. Attack" << std::endl;
+	std::cout << "2. dodge" << std::endl;
+	std::cout << "3. flee" << std::endl;
+
+	utils::GetUserInput(tutorial_choice);
+	std::cout << std::endl;
+	while (tutorial_choice != 2)
+	{
+		std::cout << "Invalid choice! Please choose again: " << std::endl;
+		utils::GetUserInput(tutorial_choice);
+	}
+
+	std::cout << "The main character rolled 15" << std::endl;
+	std::cout
+		<< "Main character takes a dodge action! the enemy has a disadvantage "
+		<< std::endl;
+	std::cout << "Minotaur attacks!" << std::endl;
+	std::cout << "Minotaur rolled 8" << std::endl;
+	std::cout << "The main character has better AC than the attack of the "
+				 "Minotaur, Minotaur misses!";
+	std::cout << std::endl;
+
+	std::cout << "Please select the flee option:" << std::endl;
+	std::cout << "1. Attack" << std::endl;
+	std::cout << "2. dodge" << std::endl;
+	std::cout << "3. flee" << std::endl;
+
+	utils::GetUserInput(tutorial_choice);
+	std::cout << std::endl;
+	while (tutorial_choice != 3)
+	{
+		std::cout << "Invalid choice! Please choose again: " << std::endl;
+		utils::GetUserInput(tutorial_choice);
+	}
+	std::cout << "The main character rolled 9" << std::endl;
```

Comment:
Change line to something more descriptive, since we have the ability (also, fleeing does not have any roll involved):
"You look around to find an opening to flee, but find none in this locked up arena."